terragrunt/modules/crates-io/cloudfront-webapp.tf

```diff
@@ -94,6 +94,12 @@ resource "aws_cloudfront_distribution" "webapp" {
     }
   }
 
+  logging_config {
+    include_cookies = false
+    bucket          = aws_s3_bucket.logs.bucket_regional_domain_name
+    prefix          = "cloudfront/${var.webapp_domain_name}"
```

Comment:
Do we want to log all requests or just `static.crates.io`? AFAIK requests to the app should be logged already.

Reply:
The application itself already has request logs in Papertrail, those are probably fine for that purpose. I think just logging static.crates.io will be enough for now.

(crates.io itself serves comparatively little bandwidth, though most static.crates.io requests are redirects from crates.io I believe).

Reply:
(But it might make this more complicated to limit the logging -- logging all requests seems also fine to me).

Reply:
I've removed logging from the other two CloudFront distributions and left it untouched for `static.*`.